extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -126,6 +125,11 @@ export class AmpUserNotification extends AMP.BaseElement {
       assertHttpsUrl(this.dismissHref_, this.element);
     }
 
+    const alwaysShowAttr = this.element.getAttribute('always-show');
+    /** @private @const {boolean} */
+    this.alwaysShow_ = (this.element.hasAttribute('always-show') &&
+        alwaysShowAttr != '0' && alwaysShowAttr != 'false');
```

Comment:
qq, do we allow a value? so this means any value is valid except 0 and false? (can we be more consistent with that)